lib/schema/basal.js

```diff
@@ -139,6 +139,20 @@ module.exports = function(streamDAO){
     'basal',
     'deliveryType',
     [
+      schema.makeHandler(
+        'automated',
+        {
+          schema: {
+            deviceTime: schema.validDeviceTime,
+            scheduleName: schema.ifExists(schema.isString),
+            rate: schema.isNumber,
+            duration: schema.ifExists(schema.and(schema.isNumber, schema.greaterThanEq(0))),
+            previous: schema.ifExists(schema.or(schema.isObject, schema.isString)),
+            suppressed: schema.ifExists(schema.isObject)
```

Comment:
I don't think an automated basal will suppress another basal so we can probably remove this.

Reply:
(Nor should the scheduled basal below, but 🤷.)

Reply:
You don't think an automated basal suppresses the scheduled basal that would've occurred if the algorithm wasn't running? Having the suppressed allows us to show this:
![image](https://user-images.githubusercontent.com/418233/108836157-6991bd80-75c8-11eb-9788-56f62ad42c3e.png)

But maybe this is not useful info?

Reply:
Ah, good point. Yes, let's leave that in. I think that is missing in platform. I added it to https://tidepool.atlassian.net/browse/BACK-1903.